DirectProgramming/DPC++FPGA/Tutorials/Features/ac_types/ac_int/README.md

```diff
@@ -56,40 +56,39 @@ The tutorial consists of several functions, each of which contains a SYCL kernel
 
 #### Basic Operations and Promotion Rules
 
-When using ac_int, we can write Addition, Division, Multiplication operations to use precisely as many bits as are needed to store the results. This is demonstrated by the kernels `Add`, `Div` and `Mult`.
+When using `ac_int`, we can write Addition, Division, Multiplication operations to use precisely as many bits as are needed to store the results. This is demonstrated by the kernels `Add`, `Div` and `Mult`.
 
-ac_int automatically promotes the result of all operations to the number of bits needed to represent all possible results without overflowing. For example, the addition of two 8-bit integers results in a 9-bit result to support overflow.
+`ac_int` automatically promotes the result of all operations to the number of bits needed to represent all possible results without overflowing. For example, the addition of two 8-bit integers results in a 9-bit result to support overflow.
 
-However, if the user attempts to store the result in an 8-bit container, ac_int will let the user do this, but this leads to the discard of the extra carry bit. The responsibility lies on the user to use the correct datatype.
+However, if the user attempts to store the result in an 8-bit container, `ac_int` will let the user do this, but this leads to the discard of the extra carry bit. The responsibility lies on the user to use the correct datatype.
 
 These promotions rules are consistent across all architectures so the behavior should be equivalent on x86 or on FPGA.
 
 #### Shift Operation
 
-The behavior of a shift operation with ac_ints is slightly different from its behvior with native integer types. For full details, see the ac_int documentation in the file `ac_data_types_ref.pdf`. Some key points to remember are as follows:
-- If the datatype of the shift amount is signed, negative shift amounts are accepted. A shift by a negative amount is equivalent to a positive shift in the opposite direction.
-- If the user knows that the shift direction and that the shift value is always positive, it is advisable to use an unsigned datatype for the shift value to obtain better QoR.
-- Shift values of greater than the width of the datatypes are treated as a shift equal to the width of the datatype.
+The behavior of a shift operation with an `ac_int` is slightly different from its behavior with native integer types. For full details, see the `ac_int` documentation in the file `ac_data_types_ref.pdf`. Some key points to remember are as follows:
+- If the datatype of the shift amount is not explicitly unsigned (either using ac_int<N, *false*> or using the unsigned keyword), then the compiler will generate a more complex shifter that allows negative shifts and positive shifts. A right-shift by a negative amount is equivalent to a positive left-shift.
+- Normally, you will not want to enable negative shifting, so you should use an unsigned datatype for the shift value to obtain a more resource efficient design.
```

Comment:
indent

also minor nitpick, should use `ac_int` and `unsigned` :)

Reply:
Updated, thank you!